src/main/java/Driver.java

```diff
@@ -1,39 +1,66 @@
-public class Driver implements IDriver{
+import exceptions.*;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.io.IOException;
+import java.nio.file.Paths;
 
-    @Override
-    public void createDatabase(String schemaPath) {
 
+public class Driver implements IDriver {
+    private final Executer executer;
+    public Driver(final Executer executer) {
+        this.executer = executer;
     }
 
     @Override
-    public void set(String databaseName, String jsonObject) {
-
+    public void createDatabase(String schemaPath) throws IOException, JSONException {
+        String command = String.format("%s -cmd create -sch %s",Paths.get("source","main.py").toString(), schemaPath);
+        JSONObject api = executer.execute(command);
+        exceptionRaiser(api);
     }
 
     @Override
-    public String get(String databaseName, String jsonQuery) {
-        return null;
+    public void set(String databaseName, String tableName, String jsonObject) throws JSONException, IOException {
+        String command = String.format("%s -cmd set -database %s -table %s -val \"%s\"",Paths.get("source","main.py").toString(), databaseName, tableName, jsonObject);
+        JSONObject api = executer.execute(command);
+        exceptionRaiser(api);
     }
 
     @Override
-    public void delete(String databaseName, String jsonQuery) {
+    public JSONArray get(String databaseName, String tableName, String jsonQuery) throws JSONException, IOException {
+        String command = String.format("%s -cmd get -database %s -table %s -q \"%s\"",Paths.get("source","main.py").toString(),databaseName ,tableName, jsonQuery.toString());
+        JSONObject api = executer.execute(command);
+        exceptionRaiser(api);
+        String result = api.getString("result").replace("{","\"{").replace("}","}\"");
```

Comment:
Why do you need this replace?

Reply:
because in the get the python return ```"result": [{josn},{json}]``` with no quotes before the ```{``` so when i pass it without the replacement the ```JSONARRAY``` throw exceptions so I should add the quotes.